library/src/main/java/com/datatorrent/lib/simple/SingleInputMultiOutput.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.lib.simple;
```

Comment:
Not sure if I understand the usecase of SingleInputMultiOutput operator.

This seems more like a stream duplicator except with addition of processing calls.

There is a com.datatorrent.lib.stream.StreamDuplicater operator in malhar-library which splits the stream across 2 streams. Possibly one can use StreamDuplicator operator rather than using this.